### PR TITLE
Dummy TRD conversion of digits to ADC (O2-455)

### PR DESCRIPTION
**Pull request by jolopezl**
This is work in progress. Dummy implementation of TRD conversion of digits to signals and ADC counts (O2-455) and continuation of PR #1611. The purpose of this PR is to show the current status of the implementation while we prepare the classes to get access to the calibration parameters, as well as the container classes ArraySignal and ArrayADC. The digitization algorithm works, it only lacks of putting the output in the right containers.

### CODE REVIEW

File: Detectors/TRD/simulation/src/Digitizer.cxx [hunk 1]
```diff
@@ -418,6 +420,177 @@ bool Digitizer::convertHits(int det, const std::vector<o2::trd::HitType>& hits,
   return true;
 }
 
+bool Digitizer::convertSignals(const int det, int& arraySignal)
```
Comment (tdietel) [inline Detectors/TRD/simulation/src/Digitizer.cxx]: I would suggest to rename this method to convertSignalsToDigits

File: Detectors/TRD/simulation/include/TRDSimulation/Digitizer.h [hunk 2]
```diff
@@ -38,14 +38,18 @@ class Digitizer
   int mEventID = 0;
   int mSrcID = 0;
 
-  float mWion; //  Ionization potential
+  float mWion;   //  Ionization potential
+  bool mSDigits; // true :convert signals to summable digits, false by default
```
Comment (tdietel) [inline Detectors/TRD/simulation/include/TRDSimulation/Digitizer.h]: I don't see a setter for this variable.

Comment (jolopezl) [reply Detectors/TRD/simulation/include/TRDSimulation/Digitizer.h]: That variable is defined in the constructor, but I can add the setter and do it that way too.

File: Detectors/TRD/simulation/src/Digitizer.cxx (hunk 1 above)
Comment (jolopezl) [reply Detectors/TRD/simulation/src/Digitizer.cxx]: I agree, I wil rename it.

**Comment by sawenzel** [comment]
This fails with: `Digitizer.cxx:445:1: error: control may reach end of non-void function [-Werror,-Wreturn-type]
}`

**Comment by jolopezl** [comment]
Hello @sawenzel, I have fixed the issue of a non-void function without a return.

**Comment by jolopezl** [comment]
Hello @sawenzel, I think checks are failing only for timeouts not on our side. Can proceed without this checks passing?

**Pull request merged**